client/src/components/NewsWidget/NewsWidget.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ShallowRenderer from "react-test-renderer/shallow";
+import { MenuItem, Select } from "@material-ui/core";
+import NewsWidget from "./";
+import { shallow } from "enzyme";
+import { getArticles } from "./newsAPI";
+
+let component;
+
+beforeEach(() => {
+    component = shallow(<NewsWidget />);
+});
+
+test("it renders correctly", () => {
+    const shallowRenderer = new ShallowRenderer();
+    const snapshotComponent = shallowRenderer.render(<NewsWidget />);
+    expect(snapshotComponent).toMatchSnapshot();
+});
+
+test("it contains Material UI Select, MenuItem components", () => {
+    expect(component.find(Select)).toHaveLength(2);
+    expect(component.find(MenuItem)).toHaveLength(10);
+});
+
+test("dropdown default values load in", () => {
+    expect(component.find(Select).at(0).props().defaultValue).toEqual("general");
+    expect(component.find(Select).at(1).props().defaultValue).toEqual("nz");
+});
+
+test("the data is returned from the getArticles function (API endpoint works)", () => {
+    return getArticles("general", "nz").then((data) => {
+        expect(data.length).toBeGreaterThan(1);
```

Comment:
It'd be good if you could add some extra expect statements here, basically just for whatever parts of the data you're accessing in the frontend code. Since you're relying on those parts being there, it's good to include them as test conditions